python-project-template/docs/notebooks/{% if include_notebooks %}intro_notebook.ipynb{% endif %}

```diff
@@ -108,4 +108,4 @@
  },
  "nbformat": 4,
  "nbformat_minor": 5
-}
\ No newline at end of file
+}
```

Comment:
This extra return at the end seems to make the "clear output from notebooks" pre-commit happy when it is run on a brand new project.

Reply:
Personally I like files that end with a newline. Usually unnecessary but sometimes important and I have not run into a case where it does harm.